clippy_lints/src/semicolon_block.rs

```diff
@@ -0,0 +1,134 @@
+use clippy_utils::diagnostics::{multispan_sugg_with_applicability, span_lint_and_then};
+use rustc_errors::Applicability;
+use rustc_hir::{Block, Expr, ExprKind, Stmt, StmtKind};
+use rustc_lint::{LateContext, LateLintPass, LintContext};
+use rustc_session::{declare_lint_pass, declare_tool_lint};
+use rustc_span::Span;
+
+declare_clippy_lint! {
+    /// ### What it does
+    ///
+    /// Suggests moving the semicolon from a block inside of the block to its kast expression.
```

Comment:
```suggestion
    /// Suggests moving the semicolon after a block to the inside of the block, after its last expression.
```

Reply:
Sorry, I was a bit hasty on that commit I think, fixed it